src/components/VLink.vue

```diff
@@ -33,19 +31,6 @@
       rtl-flip
     />
   </a>
-  <span
-    v-else
-    v-bind="$attrs"
-    :class="{ 'inline-flex flex-row items-center gap-2': showExternalIcon }"
-  >
-    <slot /><VIcon
-      v-if="showExternalIcon && !isInternal"
-      :icon-path="externalLinkIcon"
-      class="inline-block"
-      :size="4"
-      rtl-flip
-    />
```

Comment:
Is the removal of `<VIcon/>` accidental or intentional?

Reply:
Only the Icon inside the span I'd removed, <a> and NuxtLink still have it.